library-extensions-swipe/src/main/java/com/mikepenz/fastadapter/swipe/SimpleSwipeDrawerCallback.kt

```diff
@@ -71,7 +93,13 @@ class SimpleSwipeDrawerCallback @JvmOverloads constructor(private val swipeDirs:
     }
 
     override fun onSwiped(viewHolder: RecyclerView.ViewHolder, direction: Int) {
-        // Not used
+        viewHolder.itemView.translationX = 0f
+        viewHolder.itemView.translationY = 0f
```

Comment:
Could you please describe why we need to reset the translation for this change?

Reply:
You're right, we don't need that. I just did a blind copy/paste from `SimpleSwipeCallback`. My mistake